drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
```

Comment:
In the future (https://openedx.atlassian.net/browse/SOL-1462 number 6) we will be adding `sr` descriptions to the dropped items as well, and it's important for screen reader users to be able to read the description of dropped items... but I'm not sure if they should have tabindex="0" or "-1" in that case, since they can't be interacted with.